src/components/common/forms/ContactForm.tsx

```diff
@@ -1,28 +1,47 @@
 import React from "react";
 import { useForm } from "react-hook-form";
+import { useAppDispatch } from "@/state/app/hooks";
+import { handleModalStatus, ModalStatus } from "@/state/features";
 
 import { Button, Input, Textarea } from "@components/common";
 
 import styles from "./Forms.module.css";
 
 const ContactForm = () => {
-  const { register, handleSubmit } = useForm();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const dispatch = useAppDispatch();
+  const formError = errors["email*"]?.type || errors["message*"]?.type;
+
   const onSubmit = (data: object) => {
     //on submit logic
```

Comment:
Is this comment necessary?

Reply:
Not really! It was mostly for your reference. I've erased it now